docs/content/en/docs/concepts/kube_context/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Kube-context activation"
+linkTitle: "Kube-context activation"
+weight: 80
+---
+
+This page discusses how Skaffold selects the kube-context.
+
+
+When interacting with a kubernetes cluster, Skaffold does so via a kube-context.
+Thus, the selected kube-context determines the kubernetes cluster, the kubernetes user, and the default namespace.
+By default, Skaffold uses the _current_ kube-context from your kube-config file.
+
+To override this default, Skaffold offers two options:
+
+1. Via the `kube-context` flag
+
+    ```bash
+    skaffold dev --kube-context <myrepo>
+    ```
+
+1. Via the `deploy.kubeContext` option in `skaffold.yaml`
+
+    ```yaml
+    deploy:
+      kubeContext: minikube
+    ```
+
+When both are given, the CLI flag always takes precedence.
+
+### Kube-context activation and Skaffold profiles
+
+The kube-context has a double role for Skaffold profiles:
```

Comment:
This is a little bit confusing. 
You could say, 
```suggestion
Multiple `kube-contexts` can be activated from skaffold profiles.
```

Reply:
A put some thoughts into this, if I can make this whole part clearer. Can you have another look?

Reply:
I think this makes sense. the main point here is that the kube-context has two semantic roles in the activation of skaffold profiles, but not that multiple contexts can be activated from profiles (since that's not really true).